tests/decorators/decorators.go

```diff
@@ -60,6 +60,11 @@ var (
 	RequiresNoSnapshotStorageClass = Label("RequiresNoSnapshotStorageClass")
 	// Requires a storage class with ReadWriteMany Block support
 	RequiresRWXBlock = Label("RequiresRWXBlock")
+	// Requires a storage class with ReadWriteOnce Filesystem support
```

Comment:
```suggestion
	// Requires VMStateStorageClass to be set to a ReadWriteMany Filesystem storage class
```
Would that be a more accurate comment?